### ISSUE TITLE
Gutenboarding: Links not reachable using keyboard on Firefox

### ISSUE THREAD

**Issue by razvanpapadopol** [Automattic/wp-calypso#39673]
#### What I expected
- I should be able to use TAB key to reach buttons and links on Firefox similar to Chrome

#### What happened instead
- Some buttons/links are not reachable using TAB:
  - _Back_ button (top-left)
  - _Continue/Skip_ button in Intent Gathering step
  - _Terms of Service_ link in Signup modal

#### Browser / OS version
Tested on Firefox 70 / Mac OS Catalina

#### Screenshot / Video and details
Check out comment on https://github.com/Automattic/wp-calypso/pull/39336#pullrequestreview-363158021

**Comment by simison** [comment]
I've tested that all buttons are now keyboard accessible. 👍
